cert_issuer/blockchain_handlers/ethereum/tx_utils.py

```diff
@@ -3,12 +3,20 @@
 from cert_issuer.errors import UnverifiedTransactionError
 
 
-def create_ethereum_trx(issuing_address, nonce, to_address, blockchain_bytes, gasprice, gaslimit):
+def create_ethereum_trx(nonce, to_address, blockchain_bytes, max_priority_fee_per_gas, gasprice, gaslimit):
     # the actual value transfer is 0 in the Ethereum implementation
-    from ethereum.transactions import Transaction
     value = 0
-    tx = Transaction(nonce=nonce, gasprice=gasprice, startgas=gaslimit, to=to_address, value=value,
-                     data=blockchain_bytes)
+    tx = dict(
```

Comment:
so why this change? Can't we rely on a Transaction model anymore?

Reply:
After the dict of the raw tx is created, next it is signed in `sign_transaction` like this:
`raw_tx = web3.Account.sign_transaction(transaction_to_sign, wif)`
`web3` expects either a dict or a `message_hash` (which we wouldn't want to use). Though I tried very hard to continue to use the Transaction model and found some similar data structures within the `web3` package. it seems the Transaction model just doesn't exist in the same form anymore. 
`web3.Account.sign_transaction()` and other `web3` methods validate input already anyway, so the benefits of recreating a Transaction model seem limited.

Reply:
To elaborate a bit further, the Transaction data structure is a relic of the `ethereum` library, which as you can see was shut down over 3 years ago: https://github.com/ethereum/pyethereum/

Reply:
Ok thanks for the clarification, I haven't had time to test yet